storm-client/src/jvm/org/apache/storm/topology/BaseConfigurationDeclarer.java

```diff
@@ -52,18 +52,20 @@ public T setNumTasks(Number val) {
         return addConfiguration(Config.TOPOLOGY_TASKS, val);
     }
 
+    @SuppressWarnings("unchecked")
     @Override
     public T setMemoryLoad(Number onHeap) {
         if (onHeap != null) {
```

Comment:
I agree but that is code that came before, and if we want to make that change I would prefer to do it on a separate pull request, just so a breaking change does not sneak in as part of review comments.